components/Navigation.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="flex ml-auto gap-10 m-2 mr-20 space-x-1 text-l">
```

Comment:
```suggestion
  <div class="flex ml-auto gap-2 m-2 mr-20 space-x-1 text-l">
```
Could be personal preference

Reply:
No problem! Corrected!